src/main/java/author/Author.java

```diff
@@ -0,0 +1,61 @@
+package author;
+
+import manga.Manga;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class Author {
+    private String authorName;
+    private ArrayList<Manga> mangaList;
+
+    public Author(String authorName) {
+        this.authorName = authorName;
+        this.mangaList = new ArrayList<>();
+    }
+
+    public String getAuthorName() {
+        return authorName;
+    }
+
+    public void setAuthorName(String authorName) {
+        this.authorName = authorName;
+    }
+
+    public ArrayList<Manga> getMangaList() {
+        return mangaList;
+    }
+
+    public boolean hasManga(Manga manga) {
+        return mangaList.contains(manga);
+    }
+
+    public void addManga(Manga manga) {
+        mangaList.add(manga);
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        // Check if they are the same object in memory
+        if (this == o) {
+            return true;
+        }
+
+        // Check for the correct type
+        if (o == null || getClass() != o.getClass()) {
```

Comment:
This check is unclear. Perhaps add parentheses to ensure that the check is properly processed?